web/cypress.config.ts

```
@@ -0,0 +1,136 @@
+    'OPENSHIFT_VERSION': process.env.CYPRESS_OPENSHIFT_VERSION,
+    //Specify if OPENSHIFT_LOGGING and OPENSHIFT_TRACING are enabled in this cluster.
+    'OPENSHIFT_LOGGING': process.env.CYPRESS_OPENSHIFT_LOGGING || false,
+    'OPENSHIFT_TRACING': process.env.CYPRESS_OPENSHIFT_TRACING || false, 
+  },
+  fixturesFolder: 'fixtures',
+  defaultCommandTimeout: 30000,
+  retries: {
+    runMode: 0,
+    openMode: 0,
+  },
+  viewportWidth: 1600,
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)